t/lib/OpenQA/Test/TimeLimit.pm

```diff
@@ -19,7 +19,10 @@ use Test::Most;
 sub import {
     my ($package, $limit) = @_;
     die "$package: Need argument on import, e.g. use: use OpenQA::Test::TimeLimit '42';" unless $limit;
-    $SIG{ALRM} = sub { BAIL_OUT "test exceeds runtime limit of '$limit' seconds\n" };
+    $limit *= $ENV{OPENQA_TEST_TIMEOUT_SCALE_COVER} // 3 if Devel::Cover->can('report');
+    $limit *= $ENV{OPENQA_TEST_TIMEOUT_SCALE_CI}    // 2 if $ENV{CI};
+    print "\$limit: $limit\n";
+    $SIG{ALRM} = sub { print "test exceeds runtime limit of '$limit' seconds\n" };
```

Comment:
Was the change from `print` to `BAIL_OUT` intended? I'd actually keep the `BAIL_OUT`.

Reply:
yes, definitely. Fixed.